demos/TicTacToe/Classes/TicTacToeLayer.cpp

```diff
@@ -152,7 +130,7 @@ class SampleChildListener : public firebase::database::ChildListener {
   }
 
  public:
-  // Vector of strings defining the events we saw, in order.
+  // Vector of strings that define the events we saw, in order.
```

Comment:
Vector of strings that contains the events in the order in which they occurred.

Reply:
This was copied from the quickstart database file. Should I also change in it any other instance of the SampleChildListener class?